### PR TITLE
AdoptAWidget: FittedBox

### LINKED ISSUE #69478: Adopt a Widget - FittedBox
The [FittedBox](https://master-api.flutter.dev/flutter/widgets/FittedBox-class.html) widget needs:

- A code / DartPad sample

For details about how to participate, see [flutter.dev/adoptawidget](https://flutter.dev/adoptawidget)

### PR DESCRIPTION
**Pull request by singhsuryanshu**
AdoptAWidget: FittedBox

Added a  code / DartPad sample to the FittedBox API Docs

This pull request is:

- [x] A code sample
- [ ] More references
- [ ] More explanation

closes #69478

### CODE REVIEW

**Comment by singhsuryanshu** [comment]
@googlebot I signed it!

File: packages/flutter/lib/src/widgets/basic.dart [hunk 1]
```diff
@@ -1474,6 +1474,29 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=T4Uehk3_wlY}
 ///
+///
+///{@tool sample --template=stateless_widget_scaffold_center.tmpl}
```
Comment (shihaohong) [inline packages/flutter/lib/src/widgets/basic.dart:1478]: You don't need the `.tmpl` suffix here. Also, Make sure there's at least one space between the code comment `///` and the comment contents, here and throughout the change.
```suggestion
/// {@tool sample --template=stateless_widget_scaffold_center}
```

File: packages/flutter/lib/src/widgets/basic.dart [hunk 2]
```diff
@@ -1474,6 +1474,29 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
...
+///
+///In this example, the image is stretched to fill the entire [Container](https://master-api.flutter.dev/flutter/widgets/Container-class.html) which would not happen normally without using FittedBox.
```
Comment (shihaohong) [inline packages/flutter/lib/src/widgets/basic.dart:1480]: - Make sure that each line is shorter than 80 characters if possible
- The dartdocs automatically link widgets, so you do not need to include the URL like you would in a Github comment
```suggestion
/// In this example, the image is stretched to fill the entire [Container], which would 
/// not happen normally without using FittedBox.
```

File: packages/flutter/lib/src/widgets/basic.dart [hunk 3]
```diff
@@ -1474,6 +1474,29 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
...
+///    height: 400,
+///    width: 300,
+///    color: Colors.red,
+///    child: FittedBox(
+///      child: Image.network('https://flutter.github.io/assets-for-api-docs/assets/widgets/owl-2.jpg'),
+///      fit: BoxFit.fill,
+///    ),
+///  );
+///}
+///```
```
Comment (shihaohong) [inline packages/flutter/lib/src/widgets/basic.dart:1494]:
```suggestion
/// ```dart
/// Widget build(BuildContext) {
///   return Container(
///     height: 400,
///     width: 300,
///     color: Colors.red,
///     child: FittedBox(
///       child: Image.network('https://flutter.github.io/assets-for-api-docs/assets/widgets/owl-2.jpg'),
///       fit: BoxFit.fill,
///     ),
///   );
/// }
/// ```
```

File: packages/flutter/lib/src/widgets/basic.dart [hunk 4]
```diff
@@ -1474,6 +1474,29 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
...
+///
+///
+///{@end-tool}
```
Comment (shihaohong) [inline packages/flutter/lib/src/widgets/basic.dart:1497]:
```suggestion
/// {@end-tool}
```

File: packages/flutter/lib/src/widgets/basic.dart [hunk 5]
```diff
@@ -1474,6 +1474,29 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
...
+///    color: Colors.red,
+///    child: FittedBox(
+///      child: Image.network('https://flutter.github.io/assets-for-api-docs/assets/widgets/owl-2.jpg'),
+///      fit: BoxFit.fill,
+///    ),
+///  );
+///}
+///```
+///
+///
```
Comment (shihaohong) [inline packages/flutter/lib/src/widgets/basic.dart:1496]: Remove extra line
```suggestion
```

File: packages/flutter/lib/src/widgets/basic.dart [hunk 6]
```diff
@@ -1474,6 +1474,29 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
...
+///{@end-tool}
+///
+///
```
Comment (shihaohong) [inline packages/flutter/lib/src/widgets/basic.dart:1499]: Remove extra line
```suggestion
```

**Review by shihaohong — CHANGES_REQUESTED** [review]

File: packages/flutter/lib/src/widgets/basic.dart (hunk 6 above)
Comment (singhsuryanshu) [reply packages/flutter/lib/src/widgets/basic.dart:1498]: Thanks for the feedback, I have made the changes.

File: packages/flutter/lib/src/widgets/basic.dart [hunk 7]
```diff
@@ -1474,6 +1474,28 @@ class CompositedTransformFollower extends SingleChildRenderObjectWidget {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=T4Uehk3_wlY}
 ///
+///
```
Comment (shihaohong) [inline packages/flutter/lib/src/widgets/basic.dart:1477]: nit: Just caught this extra line as well
```suggestion
```

**Review by shihaohong — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/widgets/basic.dart (hunk 6 above)
Comment (shihaohong) [reply packages/flutter/lib/src/widgets/basic.dart:1498]: I think you missed removing this extra line

**Comment by singhsuryanshu** [comment]
The checks are not running .....
@shihaohong

**Comment by shihaohong** [comment]
@singhsuryanshu The remaining checks are queued but not finished yet

**Comment by johnpryan** [comment]
@singhsuryanshu Thanks for participating in AdoptAWidget. We would like to send you a special thanks. If you are interested, please send an email to nyener at google dot com and provide a link to this issue.

**Pull request merged**